### DISCUSSION TITLE
关于数学符号正斜体的问题

### DISCUSSION THREAD

**Discussion by sirius-vincent** [sjtug/SJTUThesis#751] [提问]
打数学符号的时候，比如\Delta t前面的三角应该是正体的，但是模板中默认斜体，使用\text或者\mathrm或者\rm单独框出三角符号后，就整个空白不显示了，不知道能否解决？谢谢！

**Comment by AlexaraWu** [comment]
这个要看具体使用的数学字体。现版本下默认使用的 `newtxmath` 数学字体包提供了 `\upDelta`；如果是开启了 `unimath` 模板选项使用 `unicode-math` 包的话也可以使用 `\upDelta`，但更推荐使用专属的有限增量符号 `\increment`。
